crates/ruff_python_codegen/src/stylist.rs

```diff
@@ -35,40 +35,42 @@ impl<'a> Stylist<'a> {
         })
     }
 
-    pub fn from_tokens(tokens: &[LexResult], locator: &'a Locator<'a>) -> Self {
-        let indentation = detect_indention(tokens, locator);
+    pub fn from_program<N>(program: &Program<N>, locator: &'a Locator<'a>) -> Self {
```

Comment:
Should we just accept the tokens here instead of the entire program? It doesn't seem to need any data other than the tokens from program.